docs/examples/rx/02_listen_to_blocks/index.js

```diff
@@ -0,0 +1,25 @@
+// Import the API Rx
+const { ApiRx } = require('@polkadot/api');
+const { WsProvider } = require('@polkadot/rpc-provider');
+
+const wsProvider = new WsProvider('ws://127.0.0.1:9944');
+
+async function main () {
+  // Here we pass the (optional) provider
+  const subscriptionApiRx = ApiRx.create(wsProvider).subscribe((api) => {
+    // Use the RPC Node Interface
+    api.rpc.subscribeNewHead().subscribe((header) => {
+      console.log(`best #${header.blockNumber.toNumber()}`);
+    });
+  });
+
+  // Id of the subscription
+  console.log(`subscriptionApiRx: ${subscriptionApiRx}`);
+
+  setTimeout(() => {
```

Comment:
This will unsubscribe, blocks will stop coming it, however the app won't exit. The idea is to run until the user exits. The node will remove the subscription on exit, so no need to here.